example/constraints/all_pod_must_have_gatekeeper_namespaceselector.yaml

```diff
@@ -6,11 +6,6 @@ spec:
   match:
     kinds:
       - apiGroups: [""]
-        kinds: ["Pod"]
-    namespaceSelector:
-      matchExpressions:
-      - key: workloadtype
-        operator: In
-        values: [prodworkload]
+        kinds: ["Namespace"]
```

Comment:
why did this got updated?

Reply:
Ohh! It wasn't supposed to. I was testing somethings and committed it. Fixing it